scripts/measures/build-measures

```diff
@@ -21,6 +23,7 @@ qcdr_csv='../../../util/measures/'$currentPerformanceYear'/qcdr-measures.csv'
 qcdr_strata='../../../util/measures/'$currentPerformanceYear'/qcdr-measures-strata-names.json'
 enriched_quality_measures='../../../util/measures/'$currentPerformanceYear'/enriched-measures-data-quality.json'
 final_measures='../../../measures/'$currentPerformanceYear'/measures-data.json'
+additional_measures='../../../util/measures/'$currentPerformanceYear'/additional-measures.json'
```

Comment:
Can we rename this to manually_added_measures, or something similar? Otherwise it's confusing why there's one cahps measure by itself in an additional_measures file.

Reply:
(I thought you would be deleting this file entirely, but I see that ACO321 was added and requires it)

Reply:
Good suggestion.  I'll rename the variable and related file.